node/src/components/consensus/protocols/highway/round_success_meter/config.rs

```diff
@@ -15,9 +21,50 @@ pub(crate) const ACCELERATION_PARAMETER: u64 = 40;
 /// determined by this FTT.
 pub(crate) const THRESHOLD: u64 = 1;
 
-/// The maximum number of failures allowed among NUM_ROUNDS_TO_CONSIDER latest rounds, with which we
-/// won't increase our round length. Exceeding this threshold will mean that we should slow down.
+#[cfg(test)]
 pub(crate) const MAX_FAILED_ROUNDS: usize = NUM_ROUNDS_TO_CONSIDER - NUM_ROUNDS_SLOWDOWN - 1;
-/// The maximum number of failures with which we will attempt to accelerate (decrease the round
-/// exponent).
+#[allow(unused)]
```

Comment:
Can this be removed then?

Reply:
I thought I would keep it for the sake of having a reference but we have a method as well :+1: